src/blog/lagom-1-4-0-RC1.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Lagom 1.4.0-RC1
+date: 2017-12-22
+author_github: renatocaval
+tags: news
+summary: >
+    Lagom 1.4.0-RC1
+---
+
+It's Christmas! And we are very pleased to announce the availability of the first release candidate of Lagom 1.4.0. What a nice present to spent your time during the holiday season! :-)
+
+Contrary to the previous milestone releases, a RC is production ready. We are now entering the release candidate phase (typically 4 weeks) in which only major bugs (if any) will be fixed. 
+
+
+### Migrating from other versions
+
+ * If you are migrating into Lagom 1.4.0-RC1 from 1.4.0-M3 you should have a look at the release announcements for Lagom versions [1.3.9](/blog/lagom-1-3-9.html) and [1.3.10](/blog/lagom-1-3-10.html). The migration guide to 1.4 ([Java](/documentation/1.4.x/java/Migration14.html)/[Scala](/documentation/1.4.x/scala/Migration14.html)) has also been improved, don't forget to review it.
```

Comment:
I think this was just copied from last time? We haven't made any 1.3.x releases since M3.